git-utils/README.md

```diff
@@ -0,0 +1,14 @@
+# git-utils
```

Comment:
Would it make more sense for this to be called `test-utils` if the long term plan is to have more than just git stuff?

Reply:
Yes it would, I've updated that.